apps/crn-server/serverless.ts

```diff
@@ -240,7 +240,8 @@ const serverlessConfig: AWS = {
       },
     },
     auth0ConnectByCode: {
-      handler: './src/handlers/webhooks/webhook-connect-by-code.handler',
+      handler:
+        './src/handlers/webhooks/webhook-connect-by-code.sentryEnabledHandler',
```

Comment:
any chance you could make this more consistent ie make sure that we either always export a `sentryEnabledHandler` or just `handler`? right now some of them are called `handlers` even though they are wrapped in sentry wrappers